editor.planx.uk/src/@planx/components/ui.tsx

```diff
@@ -6,6 +6,7 @@ import CloudUpload from "@mui/icons-material/CloudUpload";
 import ContactPage from "@mui/icons-material/ContactPage";
 import CopyAll from "@mui/icons-material/CopyAll";
 import Create from "@mui/icons-material/Create";
+import DoorFrontOutlined from "@mui/icons-material/DoorFrontOutlined";
```

Comment:
nit: could [a simple folder icon](https://mui.com/material-ui/material-icons/?query=folder&selected=FolderOpen) be a more straightforward option here to communicate "this is nested inside something else"? Took me a while to make out exactly what the door was as the size it's used. 

Maybe also more in line with future "what do we call things" conversations where internal portals become "folders" and external portals become "subscribed flows" or similar!

Reply:
Yep this is a fair question!

I'm just re-using and formalising the existing icon here, as seen in the flow graph.

<img width="363" alt="image" src="https://github.com/user-attachments/assets/c666456b-5155-4cc5-afd9-c5947f0bb724">

I'll keep as-is for now, but let's re-evaluate this in line with our "naming things" chat 👍